### PR TITLE
OMIM included entries

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address https://github.com/monarch-initiative/mondo/issues/5507

close https://github.com/monarch-initiative/mondo/issues/2808

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
we can merge this PR after this one is merged
https://github.com/monarch-initiative/mondo/pull/5505

**Review by sabrinatoro — APPROVED** [review]
looks good to me. I think it can be merged.

**Pull request merged**
